internal/provider/resource_private_key.go

```diff
@@ -71,49 +90,71 @@ func resourcePrivateKey() *schema.Resource {
 			"ecdsa_curve": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: "ECDSA curve to use when generating a key",
+				Description: "Curve to use when generating an ECDSA key",
 				ForceNew:    true,
 				Default:     "P224",
 			},
 
 			"private_key_pem": {
-				Type:      schema.TypeString,
-				Computed:  true,
-				Sensitive: true,
+				Type:        schema.TypeString,
+				Description: "Private key data in PEM format",
+				Computed:    true,
+				Sensitive:   true,
+			},
+
+			"private_key_openssh": {
+				Type:        schema.TypeString,
+				Description: "Private key data in OpenSSH-compatible PEM format",
+				Computed:    true,
+				Sensitive:   true,
 			},
 
 			"public_key_pem": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Description: "Public key data in PEM format",
+				Computed:    true,
 			},
 
 			"public_key_openssh": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Description: "Public key data in OpenSSH-compatible PEM format",
+				Computed:    true,
 			},
 
 			"public_key_fingerprint_md5": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Description: "Fingerprint of the public key as MD5 hashing",
```

Comment:
```suggestion
				Description: "MD5 hash of the public key",
```

Reply:
There's a longer (Markdown) description for these attributes in the docs Attributes Reference section - might be better to copy this into the schema so it is not later obliterated by documentation generation.

Reply:
I gave a quick read to https://datatracker.ietf.org/doc/html/rfc4716#section-4, and it's kind of "reductive" to just say "MD5 hash of the public key": it's more like that you can generate fingerprinting of a key by following an algorithm. And there is the one described in RFC4716 (that does an MD5 hash of the public key), as well as new versions like the one documented in these release notes https://www.openssh.com/txt/release-6.8.

So I'm going with:
```
Fingerprint of the public key data in OpenSSH MD5 hash format
```
and
```
Fingerprint of the public key data in OpenSSH SHA256 hash format
```

Reply:
Does it make sense?

Reply:
Yes, good wording